internal/flow/service.go

```
@@ -16,12 +22,56 @@ type flowService interface {
+	if serviceUrl == "" {
+		return false, fmt.Errorf("SERVICE_URL is not set")
+	}
+	serviceUrl = strings.TrimSuffix(serviceUrl, "/")
+
+	url := fmt.Sprintf("%s/services/%s/dependencies", serviceUrl, current)
+	httpReq, err := http.NewRequestWithContext(ctx, http.MethodGet, url, nil)
+	if err != nil {
+		return false, fmt.Errorf("failed to create dependency request: %w", err)
+	}
+
+	resp, err := http.DefaultClient.Do(httpReq)
```

golangci-lint (2.12.2)

[error] response body must be closed

(bodyclose)